### PR TITLE
Made some minor changes to Sdk end to end tabular custom training notebook

### PR DESCRIPTION
**Pull request by bhupana**
Made some minor changes to Sdk end to end tabular custom training notebook

If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/CODEOWNERS) file under the `# Community Notebooks` section, pointing to the author or the author's team.
- [x] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/contributing.md#code-quality-checks).

### CODE REVIEW

File: notebooks/community/sdk/SDK_End_to_End_Tabular_Custom_Training.ipynb [hunk 1]
```diff
@@ -23,6 +23,34 @@
         "# limitations under the License."
```
Comment (andrewferlitsch) [inline notebooks/community/sdk/SDK_End_to_End_Tabular_Custom_Training.ipynb:23]: <p>remove this cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/499/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (andrewferlitsch) [inline notebooks/community/sdk/SDK_End_to_End_Tabular_Custom_Training.ipynb:23]: <p>Remove -&gt; Note: you may incur charges for training, prediction, storage or usage of other GCP products in connection with testing this SDK. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/499/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (andrewferlitsch) [inline notebooks/community/sdk/SDK_End_to_End_Tabular_Custom_Training.ipynb:23]: <p>This cell is redundant. Remove the cell.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/499/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (andrewferlitsch) [inline notebooks/community/sdk/SDK_End_to_End_Tabular_Custom_Training.ipynb:23]: <p>Vertex SDK -&gt; Vertex AI SDK</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/499/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (andrewferlitsch) [inline notebooks/community/sdk/SDK_End_to_End_Tabular_Custom_Training.ipynb:23]: <p>lowercase 'Managed'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/499/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Comment (andrewferlitsch) [inline notebooks/community/sdk/SDK_End_to_End_Tabular_Custom_Training.ipynb:23]: <p>headings need to be a verb phrase</p><p> </p><h1>Write Training Script -&gt; Write the training script</h1>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/499/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Comment (andrewferlitsch) [inline notebooks/community/sdk/SDK_End_to_End_Tabular_Custom_Training.ipynb:23]: <h1>Launch a Training Job to Create a Model -&gt; Run the training job</h1>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/499/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Comment (andrewferlitsch) [inline notebooks/community/sdk/SDK_End_to_End_Tabular_Custom_Training.ipynb:23]: <p>The docker containers should be updated to use the ones in artifact registry vs the older entries in GCR.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/499/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Comment (andrewferlitsch) [inline notebooks/community/sdk/SDK_End_to_End_Tabular_Custom_Training.ipynb:23]: <p>Deploy Model -&gt; Deploy the model</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/499/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Comment (andrewferlitsch) [inline notebooks/community/sdk/SDK_End_to_End_Tabular_Custom_Training.ipynb:23]: <h1>Predict on Endpoint -&gt; Make a prediction</h1>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/499/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Comment (andrewferlitsch) [inline notebooks/community/sdk/SDK_End_to_End_Tabular_Custom_Training.ipynb:23]: <p>remove the {TODO: ... }</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/499/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

**Comment by bhupana** [comment]
changed as per review comments

**Review by andrewferlitsch — APPROVED** [review]

**Comment by andrewferlitsch** [comment]
ingestion test not ran:
Changed files did not match file filter

**Comment by andrewferlitsch** [comment]
@ivanmkc test been running since June 27

**Comment by bhupana** [comment]
hi,

This PRis been closed ,Please find the new pr link below 

https://github.com/GoogleCloudPlatform/vertex-ai-samples/pull/1292

**Pull request closed without merging**